docs/topics/mta-default-questionnaire.adoc

```diff
@@ -5,15 +5,15 @@
 
 :_content-type: REFERENCE
 [id="mta-default-questionnaire_{context}"]
-= Default questionnaire
+= The default assessment questionnaire
 
-The default questionnaire is named **Legacy Pathfinder**. Pathfinder is a questionnaire-based tool that evaluates the suitability of applications for modernization in containers on an enterprise Kubernetes platform. 
+*Legacy Pathfinder* is the default {ProductName} ({ProductShortName}) questionnaire. Pathfinder is a questionnaire-based tool that you can use to evaluate the suitability of applications for modernization in containers on an enterprise Kubernetes platform. 
 
-Through interaction with the questionnaire and the review process, the system is enriched with application knowledge, which is exposed through a collection of reports. The reports provide information about applications suitability for Kubernetes, highlight associated risks, and generate an adoption plan informed by the applications' prioritization, business criticality and dependencies.
+Through interaction with the default questionnaire and the review process, the system is enriched with application knowledge exposed through the collection of assessment reports.
 
-The default questionnaire can be exported to YAML.
+You can export the default questionnaire to a YAML file:
 
-.To view the Legacy Pathfinder YAML, click (here).
+.The Legacy Pathfinder YAML file
```

Comment:
Suggestion : 
It's difficult to tell this is a collapsible code example. 
Could the 'The Legacy Pathfinder YAML file' header  have an arrow/triangle on its left that can be clicked to collapse/expand the code example ?

Reply:
Hmmm., seems reasonable..I can try to do something about it, but I'd actually make it non-collapsible to increase visibility. Not all our our users pay enough attention to small things and can just skip crucial info.

Reply:
Made a non-collapsible example out of it